### DISCUSSION TITLE
Is there a way to disable or remove the built in Masks?

### DISCUSSION THREAD

**Discussion by Jcontre905** [ChatGPTNextWeb/NextChat#2040] [Q&A] [answered]
Hi there

I was wondering if there was a way I could start off with a blank slate of my own prompts without the preloaded ones showing up? 

I love this Web interface other than that.

Anyone know?

**Comment by Jarrodsz** [comment]
Upvote https://github.com/Yidadaa/ChatGPT-Next-Web/issues/2043

**Comment by abicorios** [comment, marked as answer]
@Jcontre905, to remove masks, in the `app/masks/index.ts` file remove `CN_MASKS` and `EN_MASKS` usage. It will be replacing

```typescript
export const BUILTIN_MASKS: Mask[] = [...CN_MASKS, ...EN_MASKS].map((m) =>
  BUILTIN_MASK_STORE.add(m),
);
```

to

```typescript
export const BUILTIN_MASKS: Mask[] = [];
```

Also you can remove files `app/masks/cn.ts` and `app\masks\en.ts`, if need to clear more files. And remove imports of these files.

If you need also disable promts, in the `app\store\prompt.ts` file you can replace 

```typescript
let fetchPrompts = [res.en, res.cn];
```

to

```typescript
let fetchPrompts: PromptList[] = [];
```

**Reply by Jcontre905** [reply]
Awesome Thank you very kindly for your help!
